ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -217,12 +224,11 @@ def _is_ready_update(self) -> bool:
     @timed
     def _update_policy(self) -> None:
         """
-        If train_interval is met, update the SAC policy given the current reward signals.
-        If reward_signal_train_interval is met, update the reward signals from the buffer.
+        Update the SAC policy and reward signals until the steps_per_update ratio
```

Comment:
Would "until the steps_per_update ratio is met" be more appropriate as a docstring for the _update_sac_policy function?